Makefile

```diff
@@ -162,17 +162,14 @@ openrpc-gen:
 lint-imports:
 	@echo "--> Running imports linter"
 	@for file in `find . -type f -name '*.go'`; \
-		do goimports-reviser -list-diff -set-exit-status -company-prefixes "github.com/celestiaorg"  -project-name "github.com/celestiaorg/"$(PROJECTNAME)"" -output stdout $$file \
+		do goimports-reviser -list-diff -set-exit-status -company-prefixes "github.com/celestiaorg" -project-name "github.com/celestiaorg/celestia-node" -output stdout $$file \
 		 || exit 1;  \
```

Comment:
Change to `./..`?

Reply:
As far as I remember this one was changed to throw an error on first one, to help CI to fail on this check. Otherwise it was closing with exit code 0 even when some files were not formatted properly

Reply:
I don't like that this is inconsistent and would prefer to unify. But ok to keep if there is no other solution to this

Reply:
This one is different only because in this case goimports-reviser is used as linter. We expect it to return status code 1 at any errors, and it should be supported by`-set-exit-status` flag. But the flag doesn't work as internded, so we use a workaround with `find`. 

When we use it for fixing imports it works just fine with `./...`